### PR TITLE
[8.0][mail_debrand] Remove branding from mail footers.

### PR DESCRIPTION
**Pull request by yajo**
Those mail footers odoo sends that say "Sent by YourCompany using Odoo" now say nothing of Odoo.

@rafaelbn

### CODE REVIEW

File: mail_debrand/README.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+{module_title}
```
Comment (pedrobaeza) [inline mail_debrand/README.rst]: Complete the README properly

File: mail_debrand/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Module name",
```
Comment (pedrobaeza) [inline mail_debrand/__openerp__.py]: Put name and summary properly

**Comment by yajo** [comment]
Sorry, fixed. I'm about to push the Spanish translation too (before you ask :wink: )

**Comment by yajo** [comment]
Ready to review.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by rafaelbn** [comment]
:+1:

**Comment by andhit-r** [comment]
:+1

**Comment by serpentcs-dev1** [comment]
@Yajo @rafaelbn I am not sure how the method **_get_signature_footer** is triggered, method can be find here: https://github.com/OCA/social/pull/51/files#diff-c45b7b8d77d447f7aff18f69c2efc94bR12

There is a method similar to this in odoo base mail module v8 : https://github.com/odoo/odoo/blob/8.0/addons/mail/mail_followers.py#L138 

but you are not overriding the base method, neither it is called from anywhere else. So I am not sure how your method is triggered and the debranding functionality is achieved. 

Can you please explain?

cc @jbeficent

**Comment by pedrobaeza** [comment]
This module overwrites that standard method.

**Comment by serpentcs-dev1** [comment]
@pedrobaeza  Well I guess then there's a typo.

There's no underscore in front of the method name.

**Comment by pedrobaeza** [comment]
Well, AFAIK it was working. Check the same for v9 to see.

**Comment by serpentcs-dev1** [comment]
@pedrobaeza I see, okay I'll check it, thanks!

Actually, it is still needed to be migrated to v9 and I'm working on it.

**Pull request merged**